app/src/main/java/nerd/tuxmobil/fahrplan/congress/utils/FahrplanMisc.kt

```diff
@@ -0,0 +1,66 @@
+package nerd.tuxmobil.fahrplan.congress.utils
+
+import android.app.AlarmManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+
+import info.metadude.android.eventfahrplan.commons.temporal.Moment
+import nerd.tuxmobil.fahrplan.congress.MyApp
+import nerd.tuxmobil.fahrplan.congress.alarms.AlarmReceiver
+import nerd.tuxmobil.fahrplan.congress.alarms.AlarmUpdater
+import nerd.tuxmobil.fahrplan.congress.extensions.*
```

Comment:
No `*` imports please. You need to adjust your Android Studio settings to avoid this.

Reply:
Yeah, there is also some other stuff going on, for example, some of the warnings you mentioned didn't show up for me. I'm going to have to go through all of the settings sometime. Thanks for pointing this out 👍

Reply:
You can update the Kotlin import settings here:

![kotlin-imports](https://user-images.githubusercontent.com/144518/151660802-4e6b6c87-3627-4ce7-bc63-40538276db03.png)